sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def _plot_1D_results(cv_results, params, ax, xlabel, ylabel, title,
+                     fmt, xtickrotation):
+    import matplotlib.pyplot as plt
```

Comment:
i've pushed a code where I ax check is at top-most level. but now the axis check in plot_heatmap seems redundant so i was suggesting to remove that.